lib/rack/utils.rb

```diff
@@ -218,16 +218,17 @@ def select_best_encoding(available_encodings, accept_encoding)
       (encoding_candidates & available_encodings)[0]
     end
 
-    def parse_cookies(env)
-      parse_cookies_header env[HTTP_COOKIE]
-    end
-
+    # :call-seq:
+    #   parse_cookies_header(header) -> hash
```

Comment:
I personally don't think call-seq is necessary for these methods.  For example, for this method, the only thing different than the automatically generated call-seq is the return value, which is redundant considering you are documenting the return value in the text in a more descriptive manner (`map of cookie key to cookie value`).

What would be much more valuable would be adding examples for each method.  Such as for this method:

```ruby

parse_cookies_header('cookie_name=value; max-age=0')
# => {"cookie_name"=>"value", "max-age"=>"0"}
```

This should go after the text description.

Reply:
I've added the example.

I'm on the fence about removing the return type.

I think it's useful.

I can also see that adding `:call-seq:` can be overhead since any change to the method results in a change to the documentation.

If you think that this cost outweighs the benefit, or there are other reasons I'm missing, I'm happy to go through and delete all the `:call-seq:` but I think I slightly prefer it.